R/format_data.R

```diff
@@ -45,6 +45,12 @@ format_pkncaconc_data <- function(ADNCA,
     stop(paste("Missing required columns:", paste(missing_columns, collapse = ", ")))
   }
 
+  # filter out dose data if present
+  if ("PARAMCD" %in% colnames(ADNCA)) {
+    ADNCA <- ADNCA %>%
+      filter(!grepl("^DOSE", PARAMCD, ignore.case = TRUE))
```

Comment:
```suggestion
      filter(!grepl("DOSE", PARAMCD, ignore.case = TRUE))
```

I would remove the ^ as we don't know.
Also we should add, if EVID exists, filter out EVID == 1